README.md

```diff
@@ -25,6 +25,41 @@ In your app delegate's `application:didFinishLaunchingWithOptions:` method, init
 
 Tryouts SDK will automatically check for new releases every time the app becomes active and alert users when there is a new version available.
 
+### Feedback
+
+Tryouts SDK can be used to get feedbacks from users of an application. The method to be used is `presentFeedBackControllerWithPresentingViewController:animated:`
+
+```objc
+[Tryouts presentFeedBackControllerWithPresentingViewController:presentingViewController
+                                                      animated:animated];
+```
+
+Feedback overlay view will be presented on presentingViewController when this method called. Feedback overlay view has username and feedback fields, feedback is optional while username is required. Also, a snapshot of the screen is taken in the background when feedback overlay view is being presented, and sent to server attached to username and feedback data.
+
+Tryouts SDK has also support for presenting feedback view by using shake motion. This functionality is optional. To use the functionality, simply initialize your application's window with `TRYMotionRecognizingWindow` class, and implement `motionRecognizingWindowDidRecognizeShakeMotion:andTopMostController:` delegate method conforming `TRYMotionRecognizingWindowDelegate` protocol.
```

Comment:
"Tryouts SDK also has support for presenting the feedback view by using shake motion"